pandas/tests/frame/methods/test_drop_duplicates.py

```diff
@@ -333,64 +333,73 @@ def test_drop_duplicates_inplace():
     )
     # single column
     df = orig.copy()
-    df.drop_duplicates("A", inplace=True)
+    return_value = df.drop_duplicates("A", inplace=True)
     expected = orig[:2]
     result = df
     tm.assert_frame_equal(result, expected)
+    assert(return_value is None)
```

Comment:
Nit: You can remove the parenthesis from all of these `assert`s

Reply:
thx!